website/project/model.py

```diff
@@ -4588,3 +4622,36 @@ def approve(self, user):
     def reject(self, user):
         self.approval.reject(user)
         self.approval.save()
+
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
```

Comment:
I see, this class seems to be a copy/paste of [`NodeLog`](https://github.com/CenterForOpenScience/osf.io/blob/e16b8948506548011e7e617af32ffffc9428501a/website/project/model.py#L385-L585).